nixos/doc/manual/development/option-types.xml

```diff
@@ -23,30 +23,74 @@
         <literal>false</literal>.</para></listitem>
   </varlistentry>
   <varlistentry>
+    <term><varname>types.path</varname></term>
+    <listitem><para>A filesystem path, defined as anything that when coerced to 
+        a string starts with a slash. Even if derivations can be considered as 
+        path, the more specific <literal>types.package</literal> should be 
+        preferred.</para></listitem>
+  </varlistentry>
+  <varlistentry>
+    <term><varname>types.package</varname></term>
+    <listitem><para>A derivation or a store path.</para></listitem>
+  </varlistentry>
+</variablelist>
+
+<para>Integer related types:</para>
+
+<variablelist>
+  <varlistentry>
+    <term>
+      <varname>types.ints.signed</varname>
+    </term>
+    <listitem><para>A signed integer.
+    </para></listitem>
+  </varlistentry>
+  <varlistentry>
+    <term>
+      <varname>types.ints{signed8, signed16, signed32}</varname>
+    </term>
+    <listitem>
+      <para>Signed integers with a fixed length (8, 16 or 32 bits).
+        They go from
+        <inlineequation><mathphrase>−2<superscript>n</superscript>/2</mathphrase>
+        </inlineequation> to <inlineequation>
+        <mathphrase>2<superscript>n</superscript>/2−1</mathphrase>
+        </inlineequation>
+        respectively (e.g. <literal>−128</literal> to <literal>127</literal>
+        for 8 bits).
+    </para></listitem>
+  </varlistentry>
```

Comment:
Would you mind breaking these out, and specifying exactly their min/max?

Reply:
Not sure if that’d improve the description, since those are based on byte-lengths anyway.